health/micro-ui/web/micro-ui-internals/packages/modules/health-payments/src/components/attendance_inbox/filter_section.js

```
@@ -0,0 +1,136 @@
+            makeReset={() => {
+              setReset(false);
+            }}
+            isRequired={isRequired}
+            initialValue={Digit.SessionStorage.get("selectedValues")}
+            updateSeeeionStorage={(newSelectedValues) => {
+              Digit.SessionStorage.set("selectedValues", newSelectedValues);
+            }}
+            onChange={onChangeId}
+            selectedProject={projectSelected}
+            lowestLevel={Digit.SessionStorage.get("paymentConfig")?.lowestLevelBoundary || lowerBoundaryDefaultSet}
+          ></BoundaryComponent>
```

Biome (1.9.4)

[error] JSX elements without children should be marked as self-closing. In JSX, it is valid for any element to be self-closing.

Unsafe fix: Use a SelfClosingElement instead


(lint/style/useSelfClosingElements)